<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Akash-Badnale" ignored="6" name="GRP-addition" tests="8" failures="2" timestamp="20 Jul 2018 10:46:22 GMT" time="0.079" errors="0">
  <testcase name="addMethodTest" time="0.015" classname="com.ab.TestNextGeneration.CalculatorTest"/>
  <testcase name="addMethodTest" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest"/>
  <testcase name="addMethodTest" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest"/>
  <testcase name="addMethodTest" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest"/>
  <testcase name="addMethodTest2" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest"/>
  <testcase name="addMethodTest2" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest"/>
  <testcase name="addMethodTest3" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <failure type="org.testng.TestNGException" message="
Method public void com.ab.TestNextGeneration.CalculatorTest.addMethodTest3(int,double,double) requires a @DataProvider named : addMethodDataProvider3  in class com.ab.TestNextGeneration.CalculatorDataProvider">
      <![CDATA[org.testng.TestNGException: 
Method public void com.ab.TestNextGeneration.CalculatorTest.addMethodTest3(int,double,double) requires a @DataProvider named : addMethodDataProvider3  in class com.ab.TestNextGeneration.CalculatorDataProvider
at org.testng.internal.Parameters.findDataProvider(Parameters.java:513)
at org.testng.internal.Parameters.handleParameters(Parameters.java:691)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- addMethodTest3 -->
  <testcase name="addMethodTest4" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <failure type="org.testng.TestNGException" message="
Method public void com.ab.TestNextGeneration.CalculatorTest.addMethodTest4(double,int,double) requires a @DataProvider named : addMethodDataProvider4  in class com.ab.TestNextGeneration.CalculatorDataProvider">
      <![CDATA[org.testng.TestNGException: 
Method public void com.ab.TestNextGeneration.CalculatorTest.addMethodTest4(double,int,double) requires a @DataProvider named : addMethodDataProvider4  in class com.ab.TestNextGeneration.CalculatorDataProvider
at org.testng.internal.Parameters.findDataProvider(Parameters.java:513)
at org.testng.internal.Parameters.handleParameters(Parameters.java:691)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:923)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- addMethodTest4 -->
  <testcase name="multiplyMethodTest" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <ignored/>
  </testcase> <!-- multiplyMethodTest -->
  <testcase name="substractMethodTest" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <ignored/>
  </testcase> <!-- substractMethodTest -->
  <testcase name="substractMethodTest4" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <ignored/>
  </testcase> <!-- substractMethodTest4 -->
  <testcase name="substractMethodTest3" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <ignored/>
  </testcase> <!-- substractMethodTest3 -->
  <testcase name="divisionMethodTest" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <ignored/>
  </testcase> <!-- divisionMethodTest -->
  <testcase name="substractMethodTest2" time="0.0" classname="com.ab.TestNextGeneration.CalculatorTest">
    <ignored/>
  </testcase> <!-- substractMethodTest2 -->
</testsuite> <!-- GRP-addition -->
